tests/webgl/llt-three/index.html

```diff
@@ -77,22 +81,18 @@ <h2>msc_basis_transcoder test</h2>
         [THREE.RGB_ETC1_Format]: "RGB_ETC1"
```

Comment:
Can we add THREE.RGB_ETC2_Format?

Reply:
Done – although the latest threejs release doesn't have ETC2 support yet, so (regardless of this change) the demo will likely fail on Android for now. I expect the ETC2 support will be released at the end of February.